### PR TITLE
64-bit offset parameters for post kernels

### PR DESCRIPTION
**Pull request by AlexBrownAMD**

### CODE REVIEW

**Comment by nakajee** [comment]
I am wondering why we could not find this issue in our side when submitting the previous change.
Is that because we do not have any test for GSU+offsetC/D?
Maybe, it is better to add a test for that.

**Review by TorreZuk — APPROVED** [review]
Looks good to me. Only note is these are signed on rocblas side originally but I know all your math is currently unsigned so fine.

**Comment by AlexBrownAMD** [comment]
> Looks good to me. Only note is these are signed on rocblas side originally but I know all your math is currently unsigned so fine.

Yea planning to convert to signed types in a future release

**Comment by nakajee** [comment]
clang format check failed?

**Comment by AlexBrownAMD** [comment]
> I am wondering why we could not find this issue on our side when submitting the previous change. Is that because we do not have any test for GSU+offsetC/D? Maybe, it is better to add a test for that.

Yea, we have a few tests that use BufferOffset, but all offsets are relatively small and I don't think any of the current ones use GSU settings. I'll try and put together a test case that stays within RAM limits.

File: Tensile/Tests/pre_checkin/mfma/sgemm_64bit_offset_post.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+        - EdgeType: ["ShiftPtr"]
+        - PrefetchLocalRead: [True]
+      ForkParameters:
+        - AssertMinApproxSize: [0]
+        - DepthU: [8]
+        - ExpandPointerSwap: [0]
+        - GroupLoadStore: [1]
+        - GlobalSplitU: [8]
```
[the remark this answers is not in the document]
Comment (AlexBrownAMD) [reply Tensile/Tests/pre_checkin/mfma/sgemm_64bit_offset_post.yaml:36]: It was trigger a pre kernel with a 64-bit offset, but I've updated it to multibuffer so it hits a post kernel with 64-bit offset instead

**Review by yoichiyoshida — APPROVED** [review]

**Review by nakajee — APPROVED** [review]
Thanks for your update.
I do not have further comment.
Please make sure all CI tests pass with this change.

**Comment by nakajee** [comment]
Why some precheckin tests are failing?
Any missing code change?

**Comment by AlexBrownAMD** [comment]
> Why some precheckin tests are failing? Any missing code change?

The logs show all tests passing, but the job is marked fail for some reason. I've messaged Eiden about it.

**Comment by nakajee** [comment]
> > Why some precheckin tests are failing? Any missing code change?
> 
> The logs show all tests passing, but the job is marked fail for some reason. I've messaged Eiden about it.

But, I can see 9 tests failed in the log.


[  FAILED  ] 9 tests, listed below:

[  FAILED  ] HipSolutionAdapterTest.BetaOnlyKernel_Zero

[  FAILED  ] HipSolutionAdapterTest.BetaOnlyKernel_Nonzero

[  FAILED  ] OclSolutionAdapterTest.BetaOnlyKernel_Zero_Manual_Embedded

[  FAILED  ] OclSolutionAdapterTest.BetaOnlyKernel_Zero_Default_Embedded

[  FAILED  ] OclSolutionAdapterTest.BetaOnlyKernel_Zero_Default_NonEmbedded

[  FAILED  ] OclSolutionAdapterTest.BetaOnlyKernel_Nonzero

[  FAILED  ] OclSolutionAdapterTest.PlayNiceWithHip

[  FAILED  ] OclSolutionAdapterTest.TimingSingle

[  FAILED  ] OclSolutionAdapterTest.TimingMulti



 9 FAILED TESTS

**Comment by AlexBrownAMD** [comment]
> But, I can see 9 tests failed in the log.

Oh thanks, was looking at the wrong log. Errors were in HostLibraryTests. Just posted a fix. Tested locally and they're passing.

**Comment by nakajee** [comment]
> > But, I can see 9 tests failed in the log.
> 
> Oh thanks, was looking at the wrong log. Errors were in HostLibraryTests. Just posted a fix. Tested locally and they're passing.

Thanks for your quick fix.
I do not have further comments.

**Pull request merged**